### ISSUE TITLE
Tables are not converted properly

### ISSUE THREAD

**Issue by GilEliyahu23** [wouterken/htmltoadf#10]
Given that we have this HTML

```HTML
<div class='table-wrap'>
	<table class='confluenceTable'>
		<tbody>
			<tr>
				<td class='confluenceTd'>A</td>
				<td class='confluenceTd'>B</td>
				<td class='confluenceTd'>C</td>
			</tr>
			<tr>
				<td class='confluenceTd'>value 1</td>
				<td class='confluenceTd'></td>
				<td class='confluenceTd'>value 2</td>
			</tr>
		</tbody>
	</table>
</div>
```

The output is:
```json
{
	"version": 1,
	"type": "doc",
	"content": [
		{
			"type": "paragraph",
			"content": [
				{
					"type": "text",
					"text": " "
				}
			]
		},
		{
			"type": "table",
			"content": [
				{
					"type": "tableRow",
					"content": [
						{
							"type": "tableCell",
							"content": [
								{
									"type": "paragraph",
									"content": [
										{
											"type": "text",
											"text": "A"
										}
									]
								}
							]
						},
						{
							"type": "tableCell",
							"content": [
								{
									"type": "paragraph",
									"content": [
										{
											"type": "text",
											"text": "B"
										}
									]
								}
							]
						},
						{
							"type": "tableCell",
							"content": [
								{
									"type": "paragraph",
									"content": [
										{
											"type": "text",
											"text": "C"
										}
									]
								}
							]
						}
					]
				},
				{
					"type": "tableRow",
					"content": [
						{
							"type": "tableCell",
							"content": [
								{
									"type": "paragraph",
									"content": [
										{
											"type": "text",
											"text": "value 1"
										}
									]
								}
							]
						},
						{
							"type": "tableCell",
							"content": [
								{
									"type": "paragraph",
									"content": [
										{
											"type": "text",
											"text": "value 2"
										}
									]
								}
							]
						}
					]
				}
			]
		},
		{
			"type": "paragraph",
			"content": [
				{
					"type": "text",
					"text": " "
				}
			]
		}
	]
}
```

The result is wrong. the value 3 is moved to the second col. There is a missing `tableCell` node in the generated ADF.
The rendered result is:
![image](https://github.com/wouterken/htmltoadf/assets/133133353/fd5101af-2cb3-412c-b8e7-ebf45af543c5)

(Used https://developer.atlassian.com/cloud/jira/platform/apis/document/playground/ to render it)

In addition, there is a second case, when the HTML is:

```HTML
<table style="width:100%">
	<tbody>
		<tr>
			<td style="width:33.3333%">A</td>
			<td style="width:33.3333%">B</td>
			<td style="width:33.3333%">C</td>
		</tr>
		<tr>
			<td style="width:33.3333%">value 1</td>
			<td style="width:33.3333%">
				<br/>
			</td>
			<td style="width:33.3333%">value 2</td>
		</tr>
	</tbody>
</table>
```

In the following case, the generated ADF is:
```JSON
{
	"version": 1,
	"type": "doc",
	"content": [
		{
			"type": "table",
			"content": [
				{
					"type": "tableRow",
					"content": [
						{
							"type": "tableCell",
							"content": [
								{
									"type": "paragraph",
									"content": [
										{
											"type": "text",
											"text": "A"
										}
									]
								}
							]
						},
						{
							"type": "tableCell",
							"content": [
								{
									"type": "paragraph",
									"content": [
										{
											"type": "text",
											"text": "B"
										}
									]
								}
							]
						},
						{
							"type": "tableCell",
							"content": [
								{
									"type": "paragraph",
									"content": [
										{
											"type": "text",
											"text": "C"
										}
									]
								}
							]
						}
					]
				},
				{
					"type": "tableRow",
					"content": [
						{
							"type": "tableCell",
							"content": [
								{
									"type": "paragraph",
									"content": [
										{
											"type": "text",
											"text": "value 1"
										}
									]
								}
							]
						},
						{
							"type": "tableCell",
							"content": [
								{
									"type": "paragraph",
									"content": [
										{
											"type": "hardBreak"
										}
									]
								}
							]
						}
					]
				}
			]
		},
		{
			"type": "table",
			"content": [
				{
					"type": "tableRow",
					"content": [
						{
							"type": "tableCell",
							"content": [
								{
									"type": "paragraph",
									"content": [
										{
											"type": "text",
											"text": "value 2"
										}
									]
								}
							]
						}
					]
				}
			]
		}
	]
}
```

And rendered result is:
![image](https://github.com/wouterken/htmltoadf/assets/133133353/6138c621-9ed4-4e13-b104-5c20937fe786)

The table was split.

The installed package version is 0.1.9 and I am using this solution https://github.com/wouterken/htmltoadf/issues/3 
to run the code from NodeJS.

I will appreciate your help.
Thank you!

**Comment by GilEliyahu23** [comment]
While checking the behavior of the bug more, I noticed 2 main issues:

1. `<br/>` split the table. unless you wrap it under a `p` tag. so a workaround is to ensure that any column value is inside a `p` tag
2. Empty column is ignored. the first workaround will not help - having an empty `p` tag will still cause the library to ignore it (causing the second issue that `value 2` moved to `B`). ugly workaround is to have `&nbsp;` inside a `p` tag (so it will be `<p>&nbsp;</p>`


I hope that some of the information helpful.

**Comment by wouterken** [comment]
Hi @GilEliyahu23 

Could you please confirm whether the newly released version 0.1.10 addresses your issue?

**Comment by GilEliyahu23** [comment]
@wouterken 
So far it works much better.
I keep testing it.
Thank you!
